scripts/actor/actor-wfrp4e.js

```diff
@@ -1148,8 +1148,18 @@ class ActorWfrp4e extends Actor {
       cardOptions.isOpposedTest = true
     }
 
-    let dieAmount = WFRP4E.earningValues[testData.income.tier][0] // b, s, or g maps to 2d10, 1d10, or 1 respectively (takes the first letter)
-    dieAmount = Number(dieAmount) * testData.income.standing;     // Multilpy that first letter by your standing (Brass 4 = 8d10 pennies)
+    let status = testData.income.value.split(' ')
+    let tier = ''
+    let standing = status[1]
+    if(status[0] == "Silver")
+        tier = "s";
+    else if(status[0] == "Gold")
+        tier = "g";
+    else
+        tier = "b"
+
+    let dieAmount = WFRP4E.earningValues[tier][0] // b, s, or g maps to 2d10, 1d10, or 1 respectively (takes the first letter)
+    dieAmount = Number(dieAmount) * standing;     // Multilpy that first letter by your standing (Brass 4 = 8d10 pennies)
```

Comment:
You'll need to localize this.

Also I recommend you use WFRP_Utility.findkey to retrieve status value -> status key
for reference.
```
WFRP4E.statusTiers = {
    "g": "TIER.Gold",
    "s": "TIER.Silver",
    "b": "TIER.Brass"
};
```

Reply:
Got it fixed!